src/bin/jcli_app/rest/v0/node/stats.rs

```diff
@@ -25,8 +31,8 @@ impl Stats {
         let builder = reqwest::Client::new().get(url);
         let response = RestApiSender::new(builder, &debug).send().unwrap();
         response.response().error_for_status_ref().unwrap();
-        let status: serde_json::Value = response.body().json().unwrap();
-        let status_yaml = serde_yaml::to_string(&status).unwrap();
-        println!("{}", status_yaml);
+        let status = response.body().json_value().unwrap();
+        let formatted = output_format.format(status).unwrap();
```

Comment:
Now that we have a custom template formatter accepting templates from the user, could this actually fail?

Reply:
Of course, users can pass invalid template

Reply:
Yeah, but the JSON and YAML formatters "can't fail" in normal circumstances.

Reply:
Serde always leaves open door for failure, but we probably could rely on fact that `serde_json::Value` won't fail. But we always have TinyTemplate, which can and will fail often, whenever user makes a mistake.

Reply:
> But we also have TinyTemplate, which can and will fail often, whenever user makes a mistake.

Meaning that we might want handle the error in a graceful way here, rather than panicking.

Reply:
There's no infrastructure for graceful error handling in `rest` command yet.  There are a lot of panicking places, which will be turned into `return`s. I don't think that this PR is the right scope to fix that.